ha/util/message_bus.py

```diff
@@ -68,7 +68,7 @@ class CONSUMER_STATUS:
 class MessageBusConsumer:
 
     def __init__(self, consumer_id: int, consumer_group: str, message_type: str,
-                callback: Callable, auto_ack: bool, offset: str):
+                callback: Callable, auto_ack: bool, offset: str, timeout: int):
```

Comment:
The default value of timeout should be 0 as default behavior should be blocking.

Reply:
ok. I see that the default is set in get_consumer.